src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,21 @@
+p {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font: inherit;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
```

Comment:
You should set such things in a `normalize.scss` file, instead

Reply:
thanks!